lib/components/parallax_component.dart

```diff
@@ -78,21 +78,20 @@ class ParallaxLayer {
 
     // Number of images that can fit on the canvas plus one
     // to have something to scroll to without leaving canvas empty
-    final count = Vector2.all(1) + size.clone()
-      ..divide(_imageSize);
+    final Vector2 count = Vector2.all(1) + (size.clone()..divide(_imageSize));
 
     // Percentage of the image size that will overflow
-    final overflow = ((_imageSize.clone()..multiply(count)) - size)
+    final Vector2 overflow = ((_imageSize.clone()..multiply(count)) - size)
       ..divide(_imageSize);
 
     // Align image to correct side of the screen
-    final alignment = parallaxImage.alignment;
-    final marginX = alignment.x == 0 ? overflow.x / 2 : alignment.x;
-    final marginY = alignment.y == 0 ? overflow.y / 2 : alignment.y;
+    final Alignment alignment = parallaxImage.alignment;
+    final double marginX = alignment.x == 0 ? overflow.x / 2 : alignment.x;
+    final double marginY = alignment.y == 0 ? overflow.y / 2 : alignment.y;
     _scroll ??= Vector2(marginX, marginY);
 
     // Size of the area to paint the images on
-    final paintSize = count..multiply(_imageSize);
+    final Vector2 paintSize = count..multiply(_imageSize);
```

Comment:
do we need the explicit type for local variables? I think in general we dont do that

Reply:
It became clearer here since Offset is also used.

Reply:
After reading the style guide again now it should be removed on the places where it doesn't clarify anything though, like the explicit doubles I added.